pkg/agent/secondarynetwork/init.go

```diff
@@ -67,7 +69,7 @@ func NewController(
 	// k8s.v1.cni.cncf.io/networks Annotation defined.
 	podWatchController, err := podwatch.NewPodController(
 		k8sClient, netAttachDefClient, podInformer,
-		podUpdateSubscriber, primaryInterfaceStore, ovsBridgeClient)
+		podUpdateSubscriber, primaryInterfaceStore, ovsBridgeClient, nodeConfig)
```

Comment:
Could we move `nodeConfig` to be before `ovsBridgeClient`?

Reply:
done